pages/COVID.md

```diff
@@ -17,7 +17,7 @@ We are expecting over 600 people to join us over the course of 10 days. Communit
 ## What to do while attending the workshop
 
 * **Masking is required in all indoor meeting rooms.** Speakers are not required to wear masks while speaking.
```

Comment:
"Masking is required..." -> "Masking with a well-fitted, high-quality (N95, FF94, or surgical) mask is required..."
"meeting rooms" -> "spaces" to make it clear this includes hallways, lobbies, etc.
I suggest we do not make an exception for speakers.